### PR TITLE
Remove double quotes from the username and password in the example co…

### PR DESCRIPTION
**Pull request by benseven**
…nfiguration

I just spent an hour trying to work out what I was getting wrong with adding Evohome to HA. If there's no quotes needed around the username and password strings, there shouldn't be quotes around them in the example code.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/evohome.markdown [hunk 1]
```diff
@@ -34,10 +34,12 @@ To set up this integration, add the following to your `configuration.yaml` file:
 ```yaml
 # Example configuration.yaml entry
 evohome:
-  username: "YOUR_USERNAME"
-  password: "YOUR_PASSWORD"
+  username: YOUR_USERNAME
+  password: YOUR_PASSWORD
```
Comment (frenck) [inline source/_integrations/evohome.markdown:38]: These changes are not correct. String should be enquotes, as per our YAML documentation standards.

../Frenck

**Comment by frenck** [comment]
Thanks for your PR. The current documentation, is however, correct.

../Frenck

**Comment by benseven** [comment]
With quotes, the configuration does not work.

Without quotes, the configuration works.

If you haven't tested that - as I have this morning, with a now working Evhohome integration based on these changes and these changes alone - then you are entirely wrong to refuse this PR.

File: source/_integrations/evohome.markdown (hunk 1 above)
Comment (benseven) [reply source/_integrations/evohome.markdown:38]: Which is preferable - documentation that meets your standards, or documentation that causes the software to actually work?

**Comment by frenck** [comment]
> With quotes, the configuration does not work.
> Without quotes, the configuration works.

There is no difference in YAML's perspective, meaning you might have a different issue at hand.
If you suspect a bug, please raise an issue in our Core issue tracker.

File: source/_integrations/evohome.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/evohome.markdown:38]: If it doesn't work, then it sounds like a bug. As there is no difference in those syntaxes for the outcome. Both are considered a string by YAML. As a matter of fact, enquoting is better, as it is more explicit (hence it is the standard for documenting string values).

../Frenck

Comment (benseven) [reply source/_integrations/evohome.markdown:38]: I've retested and added the quotes back in, and it still works so this PR can be ditched. For whatever reason HA didn't pick up or start working on the first restart, but on the second restart (when I had removed the quotes) it did  - my apologies.

**Pull request closed without merging**